samples/SocialSample/Startup.cs

```diff
@@ -131,56 +133,61 @@ 127.0.0.1 MsSecSample.localhost.this
+                        response.EnsureSuccessStatusCode();
+
+                        var user = JObject.Parse(await response.Content.ReadAsStringAsync());
+                        
+                        var identifier = user.Value<string>("id");
+                        if (!string.IsNullOrEmpty(identifier))
                         {
-                            identity.AddClaim(new Claim(ClaimTypes.NameIdentifier, id, ClaimValueTypes.String, context.Options.ClaimsIssuer));
+                            notification.Identity.AddClaim(new Claim(
+                                ClaimTypes.NameIdentifier, identifier,
+                                ClaimValueTypes.String, notification.Options.ClaimsIssuer));
                         }
-                        var userName = user.TryGetValue("login", out value) ? value.ToString() : null;
+
+                        var userName = user.Value<string>("login");
                         if (!string.IsNullOrEmpty(userName))
                         {
-                            identity.AddClaim(new Claim(ClaimsIdentity.DefaultNameClaimType, userName, ClaimValueTypes.String, context.Options.ClaimsIssuer));
+                            notification.Identity.AddClaim(new Claim(
+                                ClaimsIdentity.DefaultNameClaimType, userName,
+                                ClaimValueTypes.String, notification.Options.ClaimsIssuer));
                         }
-                        var name = user.TryGetValue("name", out value) ? value.ToString() : null;
+
+                        var name = user.Value<string>("name");
                         if (!string.IsNullOrEmpty(name))
                         {
-                            identity.AddClaim(new Claim("urn:github:name", name, ClaimValueTypes.String, context.Options.ClaimsIssuer));
+                            notification.Identity.AddClaim(new Claim(
+                                "urn:github:name", name,
+                                ClaimValueTypes.String, notification.Options.ClaimsIssuer));
                         }
-                        var link = user.TryGetValue("url", out value) ? value.ToString() : null;
+
+                        var link = user.Value<string>("url");
                         if (!string.IsNullOrEmpty(link))
                         {
-                            identity.AddClaim(new Claim("urn:github:url", link, ClaimValueTypes.String, context.Options.ClaimsIssuer));
+                            notification.Identity.AddClaim(new Claim(
+                                "urn:github:url", link,
+                                ClaimValueTypes.String, notification.Options.ClaimsIssuer));
                         }
-
-                        context.Principal = new ClaimsPrincipal(identity);
                     },
                 };
             });
 
             // Choose an authentication type
-            app.Map("/login", signoutApp =>
```

Comment:
Very possible i regressed something until the ci goes green. Music store isn't green yet due to lots of issues so I will probably fix this today